.azure/vs2022.yml

```diff
@@ -1698,11 +1717,15 @@ stages:
                 -D CMAKE_BUILD_TYPE=Release
                 -D CMAKE_C_COMPILER=$(workspace.root)/toolchain-amd64/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin/clang-cl.exe
                 -D CMAKE_C_COMPILER_TARGET=$(platform)-unknown-windows-msvc
-                -D CMAKE_C_FLAGS="/GS- /Oy /Gw /Gy -Xclang -fno-split-cold-code -I $(Pipeline.Workspace)/windows-sdk-$(arch)/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/lib/swift -I $(Pipeline.Workspace)/windows-sdk-$(arch)/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/lib/swift/Block"
-                -D CMAKE_CXX_COMPILER=$(workspace.root)/toolchain-amd64/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin/clang-cl.exe
-                -D CMAKE_CXX_COMPILER_TARGET=$(platform)-unknown-windows-msvc
-                -D CMAKE_CXX_FLAGS="/GS- /Oy /Gw /Gy -Xclang -fno-split-cold-code -I $(Pipeline.Workspace)/windows-sdk-$(arch)/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/lib/swift -I $(Pipeline.Workspace)/windows-sdk-$(arch)/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/lib/swift/Block"
-                -D CMAKE_MT=mt
+          - task: PowerShell@2
+            displayName: Extract swift-syntax
+            inputs:
+              targetType: 'inline'
+              script: |
+                $module = "$(Agent.BuildDirectory)/swift-syntax-$(arch)/cmake/modules/SwiftSyntaxConfig.cmake"
+                $bindir = cygpath -m $(Agent.BuildDirectory)/swift-syntax-$(arch)
+                (Get-Content $module).Replace('<BINARY_DIR>', "${bindir}") | Set-Content $module
+
```

Comment:
Wow, I am wondering how it could even pass the build. Have to check this.

Reply:
Ah, I see. Something got tangled when I tried to split whole diff into separate commits by grabbing relevant parts from a huge diff. This got corrected in next few commits. Should I rebase everything from broken commit to fix that? Or, if we're going to squash this anyway, it is probably not worth to put effort for a minor correction.

Reply:
I'm fine with either. We can certainly squash and merge if you prefer.

Reply:
I hate rebasing long commit chains, but this time it worked without much trouble. I guess a little verbosity would not harm, because we have more detailed blame. Did move problematic part to the devtools commit and everything is in the right place now. The result diff is exactly same as it was before rebase.